assets/embauche.html

```diff
@@ -251,6 +271,22 @@
 				<th>CRDS</th>
 				<td><span data-source="crds_salaire"></span>&nbsp;€</td>
 			</tr>
+			<tr class="imprecise">
+				<th>Versement transport</th>
+				<td>
+					<span data-source="versement_transport"></span>&nbsp;€
+					<aside>
+						<p>Dépend de l'autorité organisatrice des transports de la ville.</p>
```

Comment:
> Le taux de la contribution est fixé par la commune ou le groupement de communes.

[Source](https://www.service-public.fr/professionnels-entreprises/vosdroits/F31031).

In rural zones, “ville” is misleading.